src/control/ros/Conversions.cpp

```diff
@@ -265,8 +276,29 @@ void extractTrajectoryPoint(
 
   Eigen::VectorXd tangentVector;
   auto state = space->createState();
+
+  auto prevState = space->createState();
+  space->convertPositionsToState(previousPoint, prevState);
+
   trajectory->evaluate(timeAbsolute, state);
-  space->logMap(state, tangentVector);
+  space->convertStateToPositions(state, tangentVector);
+
+  auto geodesicInterpolator
+      = std::dynamic_pointer_cast<aikido::statespace::GeodesicInterpolator>(
+          interpolator);
+  if (!geodesicInterpolator)
+  {
+    throw std::invalid_argument(
+        "The interpolator of trajectory should be a GeodesicInterpolator");
+  }
+
+  auto diff = geodesicInterpolator->getTangentVector(prevState, state);
```

Comment:
Hmm, this seems to all be part of the SO(2) and R1 conversion that was originally part of #496. [AVK and I decided at the time](https://github.com/personalrobotics/aikido/pull/496#issuecomment-463749864) to address this in rewd_controllers instead (https://github.com/personalrobotics/rewd_controllers/issues/30). Has that changed?

Reply:
@brianhou It probably hasn't. Some of these changes may be holdovers from an earlier branch. I'm actually considering completely abandoning this branch, as it seems a lot of the changes are redundant.